scanners/ffuf/integration-tests/jest.config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file necessary?

Reply:
Yeah, you're right. I think we should only include a separate jest.config.json file if it differs from the default behaviour. I created a PR remove these files here #1482